common/src/Microsoft.Azure.IIoT.Agent.Framework/src/Agent/Default/Worker.cs

```diff
@@ -79,14 +79,14 @@ public class Worker : IWorker, IDisposable {
             await _lock.WaitAsync();
             try {
                 if (_cts != null) {
-                    _logger.Warning("Worker already running");
+                    _logger.Verbose("Worker already running");
```

Comment:
>Verbose [](start = 28, length = 7)

that's more like an error condition, when it happens, we should know it happened.

Reply:
306 hits in 10 min, is it so informative that we need it? for an error condition, it happens a lot

Reply:
then we need to find out why and not hide it :)